FrontDesk/src/FrontDesk.Api/Endpoints/File/Read.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Ardalis.ApiEndpoints;
+using BlazorShared.Models;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace FrontDesk.Api.FileEndpoints
+{
+    public class Read : BaseAsyncEndpoint<string, FileItem>
+    {
+        public Read()
+        {
+        }
+
+        [HttpGet("api/files/{fileName}")]
+        [SwaggerOperation(
+            Summary = "Reads a file",
+            Description = "Reads a file",
+            OperationId = "files.read",
+            Tags = new[] { "FileEndpoints" })
+        ]
+        public override async Task<ActionResult<FileItem>> HandleAsync([FromRoute]string fileName, CancellationToken cancellationToken)
+        {
+            if (string.IsNullOrEmpty(fileName)) return BadRequest();            
+
+            var fullPath = Path.Combine(Directory.GetCurrentDirectory(), @"images/Patients", fileName);
```

Comment:
Better to use an interface for file access here and implement it separately. We might want to use S3 or Azure Blog Storage etc.